index.html

```diff
@@ -121,22 +121,22 @@ <h2>Introduction</h2>
       <em>This section is non-normative.</em>
 
       <p>
-        Welcome dear implementer of Verifiable Credentials! We understand that reading
-        and nagivating technical specifications can be confusing at times. For this reason,
-        we would like to offer you a few resources to get you started.
+Reading and navigating technical specifications can be confusing. For this
+reason we have created this guide, which provides some examples and resources
+for implementing protocols which make use of <a>verifiable credentials</a>.
       </p>
       <p>
-        First, you should probably become familiarized with the official
-        <a href="https://www.w3.org/TR/verifiable-claims-use-cases/">Use Cases document</a>,
-        which offers a concise but limited collection of use cases readers should review before
-        undestanding how Verifiable Claims can best help them.
+It may be useful to become familiarized with the official
```

Comment:
`become familiarized`
-> `first familiarize yourself`

Reply:
Thanks, fixed in [aa459c1](https://github.com/w3c/vc-imp-guide/pull/26/commits/aa459c1497ccdfa8682ca2ed8138a7b747340ff4)